SU-SOE/stanford_soe_helper
# Changing to from 7.x-2.1-alpha1 hotfix for php warning

# READY FOR REVIEW

# Summary
- Hotfix for PHP Warning. Forgot to add isset to the if statement (WM)
- Update all version numbers in .info files to 7.x-2.1-alpha2 (WM)
- Update CHANGELOG.txt with a human-readable version of what’s been changed since 7.x-2.1-alpha1 (WM)


# Needed By (04/03/2018)

# Criticality
- Client is eagerly awaiting this.

# Steps to Test
- Make sure there is not an Undefined index warning on this page: https://sites.stanford.edu/jse-soe-dev/faculty-research/departments/aeronautics-astronautics you can compare it to this page: https://sites.stanford.edu/jse-soe-stage/faculty-research/departments/aeronautics-astronautics
- Make sure all the .info files and CHANGELOG.txt files are correct with the 7.x-2.1-alpha2 update

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2782

## More Information

## Folks to notify
@minorwm, @boznik
